cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/VelocityCommandManager.java

```diff
@@ -165,13 +166,26 @@ public final boolean hasPermission(final @NonNull C sender, final @NonNull Strin
 
     /**
      * {@inheritDoc}
-     * <p>
-     * In the case of the {@link VelocityCommandManager}, Brigadier is always used for command registration,
-     * and therefore this method will never return {@code null}.
      *
+     * <p>This will always return true for {@link VelocityCommandManager}s.</p>
+     *
+     * @return {@code true}
+     * @since 2.0.0
+     */
+    @Override
+    public boolean hasBrigadierManager() {
```

Comment:
should this be final? It makes no sense to override it. Same deal wit fabric. Do we want `@API` annotations these?